src/Component/Dashboard/DashboardStatistics/DashboardStatistics.tsx

```diff
@@ -38,7 +38,7 @@ export const DashboardStatistics: React.FC<DashboardStatisticsProps> = ({
     setLoadingState('loading');
     try {
       const fetchedEntities = await service?.findAll();
-      const count = fetchedEntities.length || 0;
+      const count = fetchedEntities.content.length || 0;
```

Comment:
minor: the content size should also be available via `content.totalElements`

Reply:
Sure, fixed in 5ebd4b9.